application/controllers/admin/globalsettings.php

```diff
@@ -62,6 +62,49 @@ public function refreshAssets()
         }
     }
 
+    public function sendTestEmail()
+    {
+        //Use the current user details for the default administrator name and email for this survey
+        $user = User::model()->findByPk(Yii::app()->session['loginID']);
+        $owner = $user->attributes;
+
```

Comment:
You can temporary set emailsmtpdebug to 1 or 2, but maybe best is to directly update mailer to get all debug part.

````
$mail =  new LimeMailer(); // Get the default mailer
$mail->SMTPDebug = 2 // (or 4 unsure)
````